plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -177,12 +177,12 @@ export class JenkinsApiImpl {
 
   private static async getClient(jenkinsInfo: JenkinsInfo) {
     // The typings for the jenkins library are out of date so just cast to any
-    return jenkins({
+    return new (Jenkins as any)({
```

Comment:
Do we really need the `as any` still here? We've got `@types/jenkins` installed, so maybe they need updating?

Reply:
It doesn’t compile without it

Reply:
We are using the latest version.

> "@types/jenkins@npm:^0.23.1":
  version: 0.23.3

Reply:
types mustn't have been updated for the 1.0.0 release, https://github.com/backstage/backstage/issues/13692#issuecomment-1249593608

https://github.com/silas/node-jenkins/compare/0.28.1...1.0.1

The pre-existing comment one line above said there was already an issue with the types, I removed the `as any` from the end of the line, but had to add it the constructor to make it work

Reply:
Hmm - these types are really not even useful to have in the project at this point to be honest. Wondering if we should roll out own or maybe contribute type fixes. Happy probably get this through in the meantime, but it's a little unfortunate that they haven't been updated. :)